linode/manager
# change: [M3-7446] - VLAN availability text on Linode Create

## Description 📝
This PR refactors the list of VLAN available regions in the VLAN Linode Create flow since it was too long and hard to read. 

## Changes  🔄
- Implement new notice with tooltip containing VLAN available regions (both in legacy Panel and VPC accordion)
- Implement VPC accordion test (only since panel is about to be deprecated)

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-12-13 at 14 01 42](https://github.com/linode/manager/assets/130582365/9bdb6bb8-e12e-47ca-9de2-9b6cab454102) | ![Screenshot 2023-12-13 at 14 01 15](https://github.com/linode/manager/assets/130582365/24d1fed6-af60-4eee-9096-3c5fcd4738f5) |
| ![Screenshot 2023-12-13 at 14 02 02](https://github.com/linode/manager/assets/130582365/8d5c14b6-e0f4-4804-88a2-c570f0962ca8) | ![Screenshot 2023-12-13 at 14 01 00](https://github.com/linode/manager/assets/130582365/c84576c9-6ca4-4199-850f-b301d133a050) |

## How to test 🧪

### Verification steps 
- Pull & run code locally
- Navigate to linode/create, scroll down to the VLAN panel and confirm updated panel with copy & regions tooltip
- Turn on the VPC flag, navigate to linode/create, scroll down to the VLAN accordion and expand it, and confirm updated copy & regions tooltip

## As an Author I have considered 🤔

*Check all that apply*
